### PR TITLE
Add pycircstat

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hoechenberger** [comment]
I'm vendoring the license file with this feedstock; I've created an [upstream PR](https://github.com/circstat/pycircstat/pull/78) to include it in the sdist in future releases.

File: recipes/pycircstat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - python >=3.6
+    - pip
+    - numpy
+    - scipy
+    - pandas
+    - decorator
+    - sphinx
+    - sphinx_rtd_theme
+    - nose
+    - mock
```
Comment (hoechenberger) [inline recipes/pycircstat/meta.yaml:28]: This is not a mistake – unfortunately, the package is designed such that it needs all runtime dependencies already installed when running `pip install`.

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes,  this PR is ready for review. Thank you very much!

File: recipes/pycircstat/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/pycircstat/meta.yaml:28]: Thanks for posting the explanation before we asked. Saves us a few hours debugging this :smile:

**Comment by hoechenberger** [comment]
Thank you, @ocefpaf!

**Pull request merged**